spec/lib/evm_database_ops_spec.rb

```diff
@@ -291,10 +291,28 @@ def execute_with_file_storage(action = :backup)
           execute_with_file_storage(:restore)
         end
 
+        it "parses the dirname of the `uri` and passes that in `connect_opts`" do
```

Comment:
```suggestion
        it "parses the dirname of the 'uri' and passes that in 'connect_opts'" do
```
do you really want backticks in here?

Reply:
I always use backticks to denote variable names (pulled from markdown syntax).  It isn't executed, so it is just a standard I am using to say `this_is_code`.

Reply:
nvr mind -brain freeze